node-sdk-demo/test/workspaceApi.test.js

```diff
@@ -0,0 +1,113 @@
+import { ApiClient, WorkspacesApi } from '@signadot/signadot-sdk';
+import axios from 'axios';
+import { customAlphabet } from 'nanoid';
+import { expect } from 'chai';
+
+const nanoid = customAlphabet('1234567890abcdef', 5);
+
+let previewBaseUrl;
+const SIGNADOT_ORG = 'signadot'; // Enter your Signadot org name here
+const SIGNADOT_API_KEY = '...'; // Enter your API Key from Signadot dashboard here
+
+describe('Test Signadot SDK', () => {
+    let workspacesApi, workspaceID;
+    before(async () => {
+        return new Promise(async (resolve, reject) => {
+            try {
+                const apiClient = new ApiClient();
+                apiClient.authentications.ApiKeyAuth.apiKey = SIGNADOT_API_KEY;
+                console.log(JSON.stringify(apiClient, null, 2));
+                workspacesApi = new WorkspacesApi(apiClient);
+
+                const request = {
+                    name: `test-ws-${nanoid()}`,
```

Comment:
I'm surprised that this is no different than the JSON based payload directly sent to the API - is there no intermediate objects etc?

Reply:
I can see some of these objects do exist - for example: https://github.com/signadot/node-sdk/blob/main/src/model/WorkspaceFork.js#L26